src/engine/menus.cpp

```diff
@@ -737,6 +755,7 @@ COMMAND(0, ui_tooltip, "si");
 COMMAND(0, ui_textfield, "sisbisisi");
 COMMAND(0, ui_keyfield, "sisbissi");
 COMMAND(0, ui_editor, "siiibisss");
+COMMAND(0, ui_execute , "s");
```

Comment:
new command, no doc. Why not using execute() directly?

Reply:
I think I can actually remove this entire thing